packages/cosmwasm-stargate/src/signingcosmwasmclient.ts

```diff
@@ -103,6 +105,38 @@ export interface InstantiateOptions {
   readonly admin?: string;
 }
 
+/**
+ * The options of an .instantiate() call.
+ * All properties are optional.
+ */
+export interface Instantiate2Options {
+  readonly memo?: string;
+  /**
+   * The funds that are transferred from the sender to the newly created contract.
+   * The funds are transferred as part of the message execution after the contract address is
+   * created and before the instantiation message is executed by the contract.
+   *
+   * Only native tokens are supported.
+   */
+  readonly funds?: readonly Coin[];
+  /**
+   * A bech32 encoded address of an admin account.
+   * Caution: an admin has the privilege to upgrade a contract. If this is not desired, do not set this value.
+   */
+  readonly admin?: string;
+
+  /**
+   * salt is an arbitrary value provided by the sender. Size can be 1 to 64.
+   */
+  readonly salt?: Uint8Array;
```

Comment:
This value is required. Let's add it as a required positional argument after `    codeId: number,`